src/lib.rs

```diff
@@ -71,10 +71,33 @@ pub mod rt {
             use tokio_util::compat::TokioAsyncWriteCompatExt;//Tokio02AsyncWriteCompatExt;
 
             let tcp = TcpStream::connect(self.config.get_addr()).await?;
+            tcp.set_nodelay(true)?;
```

Comment:
@windoze You can drop this; the `set_nodelay` is performed in the next line `(self.modify_tcp_stream)(&tcp)?;`. Please do so for the line in the `async_std` impl as well.